VersionControlUI/src/commands/CDiff.cpp

```diff
@@ -119,8 +119,8 @@ Interaction::CommandResult* CDiff::execute(Visualization::Item* source, Visualiz
 	versionB = unambigousPrefixPerRevision_.value(versionB, versionB);
 	QList<Model::SymbolMatcher> symbolMatcherPriorityList;
 
-	symbolMatcherPriorityList.append(Model::SymbolMatcher{"Class"});
 	symbolMatcherPriorityList.append(Model::SymbolMatcher{"Method"});
+	symbolMatcherPriorityList.append(Model::SymbolMatcher{"Class"});
```

Comment:
make all of this nicer and one one line:
`QList<Model::SymbolMatcher> symbolMatcherPriorityList{"Method", "Class"};`